src/wily/helper/output.py

```diff
@@ -0,0 +1,20 @@
+"""Output helpers for wily."""
+import json
+from collections.abc import Sequence
+
+
+def print_json(data: Sequence, headers: Sequence, path: str = ""):
```

Comment:
I think we could be more precise about the types needed here, what do you think ? 

```python
def print_json(data: Sequence[], headers: Sequence[Tuple, ...], path: str = ""):
```

I'm not sure about data's type tho

Reply:
Thank you for reviewing this!
I've tightened the typing info to List and Tuple, should be both more precise and correct.